app/forms/InstitutionSelectAddressFormProvider.scala

```diff
@@ -16,17 +16,16 @@
 
 package forms
 
-import javax.inject.Inject
-
 import forms.mappings.Mappings
 import play.api.data.Form
-import models.InstitutionSelectAddress
+
+import javax.inject.Inject
 
 class InstitutionSelectAddressFormProvider @Inject() extends Mappings {
 
-  def apply(): Form[InstitutionSelectAddress] =
+  def apply(): Form[String] =
```

Comment:
Should this form take the Address type from models if it has multiple lines of information, or will it always be a single line string?

Reply:
I believe this will always be a single line string but will be good to test this when 2707 is done to double check